package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "neomad.org",
+  "description": "Neomad is a open-source community project for nomad workers and people willing to _nomadize_ themselves.",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
```

Comment:
Ok, I think I got it. [Here you set the `watch` to true](https://github.com/neomad-team/neomad.org/pull/88/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR10).
That means that file is appropriate for dev, but not prod I guess.
Either you need to make 2 webpack config files (one for dev, one for prod), or you add a _watch_ option in your npm script (I'm not sure if that's an option?).